.drone.yml

```diff
@@ -0,0 +1,40 @@
+build:
+  image: ubuntu:xenial
```

Comment:
Reading the docs, yes.  It would look something like:

```yml
matrix:
    BASE_IMAGE:
       - ubuntu:xenial
       - ubuntu:14.04

build:
   image: $$BASE_IMAGE
```